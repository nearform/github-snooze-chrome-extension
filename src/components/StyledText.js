import React from 'react'

function StyledText({ text, color, backgroundColor }) {
  return (
    <span
      style={{ color, backgroundColor, padding: '3px', borderRadius: '25px' }}
    >
      {text}
    </span>
  )
}

export default StyledText
