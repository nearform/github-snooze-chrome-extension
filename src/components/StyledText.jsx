import { styled } from '@mui/system'
import { Typography } from '@mui/material'

const StyledText = styled(Typography)(({ theme }) => {
  return {
    color: theme.palette.primary.main,
    backgroundColor: theme.palette.secondary.main,
    padding: '5px',
    borderRadius: '25px'
  }
})

export default StyledText
