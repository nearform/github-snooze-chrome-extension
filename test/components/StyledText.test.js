import React from 'react'
import renderer from 'react-test-renderer'
import StyledText from '../../src/components/StyledText'

const props = {
  children: 'text',
  color: '#000000',
  backgroundColor: '#FFFFFF'
}

describe('StyledText.js', () => {
  test('shows the proper StyledText', () => {
    const tree = renderer.create(<StyledText {...props} />)

    expect(tree.toJSON()).toMatchSnapshot()
  })
})
