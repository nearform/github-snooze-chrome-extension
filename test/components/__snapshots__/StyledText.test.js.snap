// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledText.js shows the proper StyledText 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body1 css-1w94e69-MuiTypography-root"
  >
    text
  </p>
</DocumentFragment>
`;
